### PR TITLE
Fix misspell in docs for http.HTTPMethod (introduced in 3.11)

### PR DESCRIPTION
**Pull request by astromatt**

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Looks good, but you'll have to sign the CLA.

**Comment by AstroMatt** [comment]
CLA Signed.

**Review by AA-Turner — APPROVED** [review]

**Comment by AA-Turner** [comment]
Looks good, verified locally.

A

File: Doc/library/http.rst [hunk 1]
```diff
@@ -171,16 +171,25 @@ Property             Indicates that           Details
...
+      >>> HTTPMethod.GET
+      <HTTPMethod.GET>
+      >>> HTTPMethod.GET == 'GET'
       True
-      >>> HTTMethod.GET.value
+      >>> HTTPMethod.GET.value
       'GET'
-      >>> HTTMethod.GET.description
+      >>> HTTPMethod.GET.description
       'Transfer a current representation of the target resource.'
```
Comment (JelleZijlstra) [inline Doc/library/http.rst]:
```suggestion
      'Retrieve the target.'
```
https://github.com/python/cpython/blob/57be5459593bbd09583317ebdafc4d58ae51dbf4/Lib/http/__init__.py#L194

It's also this way on the 3.11 branch

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**